recipe/meta.yaml

```diff
@@ -1,15 +1,13 @@
-{% set name = "qtwebkit" %}
 {% set version = "5.6.2" %}
-{% set md5 = "5196f47b75e2d6cc9663c89cf9902ef0" %}
+{% set ver = version[:3] %}
```

Comment:
Change this to `'.'.join(version.split('.')[:2])`. Looks ugly, but does the job when there's versions like `5.10.0` in the future